sdk/textanalytics/Azure.AI.TextAnalytics/src/Generated/Models/DetectedLanguageInternal.cs

```diff
@@ -14,7 +14,7 @@ namespace Azure.AI.TextAnalytics.Models
         /// <param name="name"> Long name of a detected language (e.g. English, French). </param>
         /// <param name="iso6391Name"> A two letter representation of the detected language according to the ISO 639-1 standard (e.g. en, fr). </param>
         /// <param name="confidenceScore"> A confidence score between 0 and 1. Scores close to 1 indicate 100% certainty that the identified language is true. </param>
-        internal DetectedLanguageInternal(string name, string iso6391Name, double confidenceScore)
+        public DetectedLanguageInternal(string name, string iso6391Name, double confidenceScore)
```

Comment:
curious, why the need to make this class public? also, did you make the class `public` directly here? or in another file that I missed?

Reply:
I think this should be reverted